src/app/loadout-builder/generated-sets/GeneratedSet.tsx

```diff
@@ -50,18 +52,35 @@ function GeneratedSet({
     return null;
   }
 
-  const assignedMods = $featureFlags.armor2ModPicker
+  const [assignedMods] = $featureFlags.armor2ModPicker
     ? assignModsToArmorSet(
         set.armor.map((items) => items[0]),
         lockedArmor2Mods
       )
-    : {};
+    : [{}];
+
+  const canCompareLoadouts =
+    set.armor.every((items) => items[0].classType === selectedStore?.classType) &&
+    loadouts.some((l) => l.classType === selectedStore?.classType);
```

Comment:
If this is all you need `loadouts` for, consider passing in `loadoutsExistForClass` or something instead.

Reply:
Yeah can do, I was preempting that showing if a set is already a loadout ticket tbh.